src/sagemaker/clarify.py

```diff
@@ -336,6 +350,7 @@ def __init__(
         env=None,
         tags=None,
         network_config=None,
+        version="1.0",
```

Comment:
Why are we defaulting the version here? We are trying to move away from this pattern. Users don't update the SDK version can get stuck on old version with this pattern.

Reply:
Thanks. Changed to `None`.